R/inline_text.R

```diff
@@ -217,27 +217,40 @@ inline_text.tbl_uvregression <- inline_text.tbl_regression
 #' inline in an R markdown document.
 #'
 #' @param x object created from  \link{tbl_survival}
-#' @param strata if tbl_survival has multiple strata, name of strata of statistic to present. Default is `NULL` when tbl_survival have no specified strata.
-#' @param timepoint timepoint for which to return survival probability.
+#' @param strata if `tbl_survival` estimates are stratfied, level of the stratum
+#' report. Default is `NULL` when `tbl_survival` have no specified strata.
+#' @param time time for which to return survival probability.
 #' @param pattern statistics to return.  Uses \link[glue]{glue} formatting.
 #' Default is \code{'{surv} ({conf.level.100}\% {lower}, {upper})'}.  All columns from
 #' `.$table_body` are available to print as well as the confidence level (conf.level)
 #' @param ... not used
 #' @author Karissa Whiting
 #' @family tbl_survival
 #' @export
+#' @examples
+#' trial %>%
+#'   tbl_survival(
+#'     Surv(ttdeath, death) ~ trt,
+#'     times = c(12, 24)
+#'   ) %>%
+#'   inline_text(
+#'     strata = "Drug",
+#'     time = 12
+#'   )
 
 
 inline_text.tbl_survival <-
   function(x, strata = NULL,
-           timepoint,
-           pattern = "{surv} ({conf.level*100}% CI {lower}, {upper})") {
+           time,
+           pattern = "{surv} ({conf.level*100}% CI {lower}, {upper})",
+           ...) {
```

Comment:
i don't know why, but you have to include `...` as an input when using S3 functions